pkg/rdt/config.go

```diff
@@ -636,8 +636,8 @@ func (s partitionSet) resolveCacheIDRelative(id uint64, partitions []l3Partition
 		if numBits < minCbmBits {
 			numBits = minCbmBits
 		}
-		// Don't overflow, allocate all remaining bits to the last partition
-		if numBits > bitsAvailable || i == len(partitions)-1 {
+		// Don't overflow
+		if numBits > bitsAvailable {
```

Comment:
Wouldn't a warning here be of help for audacious users who try to do this but get it wrong ? Or alternatively a summary in the end that would also include the unallocated cache lines ? Or do we have something like that already ?

Reply:
We already have those basically in place. We already print an info message if the total allocation request for some cache ID is less than 100%. Also, we print a summary of what vs. requested vs. what was allocated when the whole resolving process is completed

Reply:
Ok, cool then.